### PR TITLE
landscape refactor

### PR DESCRIPTION
**Pull request by ultrasaurus**
This is not intended to change any of the content of @izgeri landscape PR, which is the result of a lot of great work of many SAFE WG members

In this PR, I made one commit, which is taking https://github.com/cn-security/safe/pull/119 verbatim and separating into 3 docs in a subfolder, then as a separate commit, I edited the language to fit more with README and separate docs which could be referenced from there

### CODE REVIEW

File: landscape/README.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+## Goal
+
+The [SAFE roadmap](../roadmap.md) includes describing the landscape of
+cloud-native security. We evaluated categories in the CNCF Landscape and
```
Comment (izgeri) [inline landscape/README.md]: It would be good to link CNCF Landscape in its first appearance in the doc (ie here)

File: landscape/README.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+## Landscape Process
+
+- [X] Draft [categories](categories.md)
+- [ ] Determine approach to category mapping (see below)
+- [ ] Map cloud-native tools into categories (adjusting categories as needed)
+- [ ] Validate categories and landscape with review by makers and users of
+    cloud-native security solutions, as well as partner working groups
+
+A note on how the work of mapping tools into the sub-categories may proceed:
```
Comment (izgeri) [inline landscape/README.md]: Maybe make this into a subsection, that we can then link above (where it says "see below")

File: landscape/categories.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+- **App definition and development**
+  - _Static Code Analysis_
+    - Inspecting code for OWASP vulnerabilities, pen testing, etc
+  - _Dependency analysis_
+    - Checking OS for vulnerabilities (updates available? Reports from https://www.us-cert.gov/ etc), image scanning
+    - Check for vulnerabilities in dependent libraries
+    - Check for maintenance of dependent projects => increased potential for vulnerabilities
+  - _Pipeline security testing_
+    - Tools that facilitate automated security testing in pipelines; eg for functional security tests of authn and authz,
```
Comment (izgeri) [inline landscape/categories.md]: Should this category be renamed to be clearer about what it contains (right now, just tools to automate security testing)? Should we enlarge it to also include other tools to secure CI/CD pipelines? How does supply chain management fit in?

File: landscape/categories.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+      - Firewalls, network segmentation, network policy
+  - _Service access_
+    - Service admittance & admission controllers
+      - Allowing new service instances to join existing services, and preventing insecure application instances from running.
+    - Storage access
+  - _Key and Certificate Management_
+    - Provisioning of access to required services for application instances, service accounts, orchestrator access to shared components, etc
+    - Provisioning of SSL certs for application instances
+
+- **Privacy**
```
Comment (izgeri) [inline landscape/categories.md]: Where does data sovereignty fit into in this category? For example, keeping European data siloed or preventing data with different levels of classification from mixing (at rest or in motion)

File: landscape/categories.md [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+  - _Storage Security_
+    - Data colocation and encryption at rest/motion
+    - Preventing insider access to protected data
+    - Monitoring for accidental data spills
+
+- **Provisioning**
+  - _Automation & Configuration Compliance_
+    - Compliance checkers, check platform configurations, verify private resources are not unexpectedly publicly accessible (eg S3 buckets), etc
+  - _Secure Container Registries_
+    - Image signing, etc
```
Comment (izgeri) [inline landscape/categories.md]: Should be clarified to be clear that secure container registries contain signed/verified images
Image signing / attestation should be moved up into the App Definition and Development as part of the pipeline-related sub-categories

Revise sub-category to Trusted Compute and add Secure Container Registries, self-hosted package repositories, etc as examples

**Comment by izgeri** [comment]
Noting that this is the updated PR connected to #66

File: landscape/categories.md (hunk 5 above)
Comment (ericavonb) [reply landscape/categories.md]: I was thinking about projects like this one:  [Open Cloud Integrity Technology](https://01.org/opencit)

File: landscape/categories.md (hunk 4 above)
Comment (ultrasaurus) [reply landscape/categories.md]: @dshaw and I thought data sovereignty was same as data location, so made that change

**Pull request merged**